misc/lldb_rb/commands/heap_page_command.py

```diff
@@ -0,0 +1,38 @@
+import lldb
+
+from lldb_rb.constants import *
+from lldb_rb.rb_base_command import RbBaseCommand
+
+class HeapPageCommand(RbBaseCommand):
+    program = "heap_page"
+    help_string = "prints out 'struct heap_page' for a VALUE pointer in the page"
+
+    def __init__(self, debugger, _internal_dict):
+        self.internal_dict = _internal_dict
```

Comment:
What's the purpose of `internal_dict`?

Reply:
It's a dictionary provided by the lldb api that contains all the functions and variables available to the debugging session.